pkg/logger/logger.go

```diff
@@ -195,7 +197,7 @@ func (l *Logger) PrintAuthf(username string, req *http.Request, status AuthStatu
 
 	err := l.authTemplate.Execute(l.writer, authLogMessageData{
 		Client:        client,
-		Host:          req.Host,
+		Host:          util.GetRequestHost(req),
```

Comment:
I think X-Forwarded would be the ones that makes sense to log. We should be inspecting that header when the mux determines where to route the traffic, so that's the host I would expect to be logged. It should reflect what the user put in their browser right?

Reply:
Yeah, if it exists - `X-Forwarded-Host` reflects the users browser where `Host` then reflects what the API Gateway changes the host to to work with the next hop internal proxy's `Host` based routing (IE K8s ingress pod).

I was trying to think through any implications of user or hacker setting a `X-Forwarded-Host` themselves manually... but I can't see any new issues or threat vectors that don't already exist since a client technically now could set the `Host` header to whatever they want now.